admin_guide/allocating_node_resources.adoc

```diff
@@ -101,6 +101,50 @@ status:
 ----
 ====
 
+[[system-resources-reported-by-kubelet]]
+== System resources reported by kubelet
+
+Since openshift 3.3 the kubelet reports system resources utilized by the container runtime and kubelet.
```

Comment:
The stats are reported namely kubelet as well. Wondering how to combine node and kubelet together. Need to check output of the command for openshift.